Channels/UserInterface/07b_VariantSets.md

```diff
@@ -0,0 +1,387 @@
+
+- ![Add](../../Assets/Icons/Plus01.png "[Add]") (Add)   
+  Click this button to add a defining attribute to the variant set. The *Add defining attributes* view is displayed, see [Add defining attributes](#add-defining-attributes).
+
+
+**Changeable attributes**
+
+- ![Search](../../Assets/Icons/Search.png "[Search]") (Search)   
+  Click this button to display the search bar and search for an attribute.
+
+- ![Refresh](../../Assets/Icons/Refresh01.png "[Refresh]") (Refresh)   
+  Click this button to update the list of an attribute.
+
+- ![Columns](../../Assets/Icons/Columns.png "[Columns]") Columns (x)   
+  Click this button to display the columns bar and customize the displayed columns and the order of columns in the list. The *x* indicates the number of columns that are currently displayed in the list.
+
+- ![Filter](../../Assets/Icons/Filter.png "[Filter]") Filter (x)   
+  Click this button to display the filter bar and customize the active filters. The *x* indicates the number of filters that are currently active.
+
+- [x]     
+  Select the checkbox to display the editing toolbar. If you click the checkbox in the header, all attributes in the list are selected.
+
+- [DELETE]   
+  Click this button to delete the selected attribute from the box. This button is only displayed if the checkbox of at least one attribute is selected.
+
+The list displays all changeable attributes to the variant set. All fields are read-only.
+
+- *Name*   
+  Attribute name.
+
+- *Key*   
+  Attribute key. The key is required for API access and must be system wide unique.
+
+- *Data type*   
+  Attribute data type.   
+  For detailed information about all data types, see [Data type list](../../DataHub/UserInterface/04_DataTypeList.md).
+
+- *ID*   
+  Attribute identification number. The ID number is automatically assigned by the system.
+
+- ![Add](../../Assets/Icons/Plus01.png "[Add]") (Add)   
+  Click this button to add a defining attribute to the variant set. The *Add changeable attributes* view is displayed, see [Add changeable attributes](#add-changeable-attributes).
+
+
+
+### Add defining attributes
+
+*Omni-Channel > Settings > Tab VARIANT SETS > Button Add > Select attribute set > Button Add (Defining attributes)*    
+*Omni-Channel > Settings > Tab VARIANT SETS > Select variant set > Button Add (Defining attributes)*
+
+![Defining attributes](../../Assets/Screenshots/Channels/Settings/VariantSets/AddDefiningAttributes.png "[Defining attributes]")
+
+- ![Back](../../Assets/Icons/Back02.png "[Back]") (Back)   
+  Click this button to close the *Add defining attributes* view and return to the *Edit variant set* view. All changes are rejected.
+
+- ![Search](../../Assets/Icons/Search.png "[Search]") (Search)   
+  Click this button to display the search bar and search for an attribute.
+
+- ![Refresh](../../Assets/Icons/Refresh01.png "[Refresh]") (Refresh)   
+  Click this button to update the list of attributes.
```

Comment:
Vgl. attribute list, variant set list... Wollen wir das generell (in default sentences) vereinheitlichen oder impossible mission? Wie? Ich wollte es hier nicht alles ändern, da die Texte aus PIM kommen.